### PR TITLE
CLB: 5 scripts, incl. 2 backgrounds

### LINKED ISSUE #829: Cloudkill

### LINKED ISSUE #831: Coronation of Chaos

### LINKED ISSUE #832: Criminal Past

### LINKED ISSUE #833: Crystal Dragon // Rob the Hoard

### LINKED ISSUE #834: Cultist of the Absolute

### PR DESCRIPTION
**Pull request by Suthro**
CLB: Added CLB scripts for missing cards

- closes #829 
- closes #831 
- closes #832 
- closes #833 
- closes #834 

*** Update 7/27/22, 1:47 AM ***

DMU: Added DMU scripts for newly spoiled cards

- Jaya, Fiery Negotiator
- 1/1 red Monk creature token with prowess
- Evolved Sleeper

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/cloudkill.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Name:Cloudkill
+ManaCost:4 B B
+Types:Sorcery
+A:SP$ PumpAll | ValidCards$ Creature | NumAtt$ -X | NumDef$ -X | SpellDescription$ All creatures get -X/-X until end of turn, where X is the greatest mana value of a commander you own on the battlefield or in the command zone.
+SVar:X:Count$HighestCMC_Card.IsCommander+YouOwn+inZoneBattlefield,Card.IsCommander+YouOwn+inZoneCommand
+Oracle:All creatures get -X/-X until end of turn, where X is the greatest mana value of a commander you own on the battlefield or in the command zone.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/cloudkill.txt:6]: AI:RemoveDeck:NonCommander

File: forge-gui/res/cardsfolder/upcoming/coronation_of_chaos.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Name:Coronation of Chaos
+ManaCost:2 R
+Types:Sorcery
+A:SP$ Pump | Cost$ 2 R | ValidTgts$ Creature | TargetMin$ 0 | TargetMax$ 3 | RememberTargets$ True | KW$ HIDDEN CARDNAME can't block. | IsCurse$ True | TgtPrompt$ Select target creature | SubAbility$ DBGoad | SpellDescription$ Up to three target creatures can't block this turn. Goad them. (Until your next turn, those creatures attack each combat if able and attack a player other than you if able.)
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/coronation_of_chaos.txt:4]: Remove `Cost$` and `RememberTargets$`

I'd make `TgtPrompt$ Select up to three target creatures`

File: forge-gui/res/cardsfolder/upcoming/criminal_past.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Name:Criminal Past
+ManaCost:2 B
+Types:Legendary Enchantment Background
+S:Mode$ Continuous | Affected$ Creature.IsCommander+YouOwn | AddKeyword$ Menace | AddStaticAbility$ PowerGrave | Description$ Commander creatures you own have menace and "This creature gets +X/+0, where X is the number of creature cards in your graveyard." (A creature with menace can't be blocked except by two or more creatures.)
+SVar:PowerGrave:Mode$ Continuous | Affected$ Card.Self | AddPower$ X | This creature gets +X/+0, where X is the number of creature cards in your graveyard.
+SVar:X:Count$TypeInYourYard.Creature
+DeckHas:Ability$Mill
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/criminal_past.txt:7]: instead probably:
```
DeckHints:Ability$Discard|Mill|Sacrifice
DeckHas:Ability$Graveyard
```

File: forge-gui/res/cardsfolder/upcoming/crystal_dragon_rob_the_hoard.txt [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+K:Vigilance
+AlternateMode:Adventure
+Oracle:Flying, vigilance
+
+ALTERNATE
+
+Name:Rob the Hoard
+ManaCost:1 W
+Types:Sorcery Adventure
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Hand | TgtPrompt$ Choose target artifact, enchantment, or legendary card in your graveyard | ValidTgts$ Artifact.YouCtrl,Enchantment.YouCtrl,Card.Legendary+YouCtrl | SpellDescription$ Return target artifact enchantment, or legendary card from your graveyard to your hand. (Then exile this card. You may cast the creature later from exile.)
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/crystal_dragon_rob_the_hoard.txt:15]: `YouOwn` better than `YouCtrl` in TgtPrompt

File: forge-gui/res/cardsfolder/upcoming/crystal_dragon_rob_the_hoard.txt [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+Oracle:Return target artifact or enchantment card from your graveyard to your hand. (Then exile this card. You may cast the creature later from exile.)
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/crystal_dragon_rob_the_hoard.txt:16]: This is missing the legendary stuff and some commas

**Review by Northmoc — COMMENTED** [review]
THANK YOU! Great work as usual

File: forge-gui/res/cardsfolder/upcoming/crystal_dragon_rob_the_hoard.txt (hunk 4 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/crystal_dragon_rob_the_hoard.txt:15]: in Desc: `artifact enchantment, or legendary` -> `artifact, enchantment, or legendary`

**Comment by Suthro** [comment]
@Northmoc Made the requested changes to the scripts. Lots of copy-paste artifacts I missed, thanks for the proof read 👍

**Review by tool4ever — APPROVED** [review]

**Pull request merged**